### DISCUSSION TITLE
Did you simulator work on your local machines?  - Jan 6

### DISCUSSION THREAD

**Discussion by ankushdharkar** [RealDevSquad/Centauri-React-Native-Cohort#7] [Show and tell]
Please post a screenshot here

**Comment by shubham-ghuge** [comment]
## Setup Done!
<img width="404" alt="Screenshot 2022-01-07 at 10 31 05 AM" src="https://user-images.githubusercontent.com/48974345/148494427-cbb9d38e-9038-4de1-9fdf-b68758be3eeb.png">

**Comment by shreya-mishra** [comment]
![image](https://user-images.githubusercontent.com/61110378/148498702-c6dc551a-b0b4-42a2-8a6d-d2a3ef7b50db.png)

**Comment by gnsharma** [comment]
Setup done
![Screenshot 2022-01-07 at 11 27 29 AM](https://user-images.githubusercontent.com/6500208/148499042-e28448f9-9ae8-439c-a56e-67c6a231929d.png)

**Comment by priyanshk20** [comment]
![image](https://user-images.githubusercontent.com/52736997/148499321-2a167c91-b791-4759-8c08-61cdd484f4eb.png)

Done!

**Comment by puneetsingh99** [comment]
Set complete!

<img width="1222" alt="Screenshot 2022-01-07 at 11 35 44 AM" src="https://user-images.githubusercontent.com/75030119/148499898-51f6072f-ae15-46ed-b2bf-dd041e1f8e3c.png">

**Comment by ganshul34** [comment]
Initial setup Completed
<img width="1340" alt="ReactNativeInitialSetup" src="https://user-images.githubusercontent.com/29010761/148500306-55ba0543-e18d-4326-be5f-7175d75be2b5.png">

**Comment by villdev** [comment]
React Native CLI setup with TypeScript
![rds rn setup](https://user-images.githubusercontent.com/50840962/148500336-2d680a5f-3f8c-4d64-bc6f-f7be2a944a8f.jpg)

**Comment by NaelaTarannum** [comment]
![Screenshot 2022-01-07 at 11 47 03 AM](https://user-images.githubusercontent.com/24935748/148501337-0c73d994-215b-4712-99fc-d8c66231ab82.png)

Setup done

**Comment by harishfalco** [comment]
Done with the setup
![image](https://user-images.githubusercontent.com/62054469/148501863-998d9c31-e7dc-4371-96d7-42167ca83067.png)

**Comment by nandanpandey43** [comment]
![Screenshot (14)](https://user-images.githubusercontent.com/38531527/148502106-cd366a25-27bd-43af-9748-33dd68b17a68.png)
## Done with the simulator and Tweet Counter App.

**Comment by vinayak-trivedi** [comment]
![Screenshot (92)](https://user-images.githubusercontent.com/90315175/148505531-c0d63784-bb65-4b97-945e-8106a9b50339.png)

**Comment by rohan09-raj** [comment]
![Screenshot (22)](https://user-images.githubusercontent.com/78433013/148507449-acc75559-ff74-4f41-8a29-20b3001ba078.png)

**Comment by nikmish1** [comment]
![hello-world-rn](https://user-images.githubusercontent.com/30014840/148507510-63b12bd4-e5d5-401e-b71b-d75e69ae4a30.png)

**Comment by PoojeshShetty** [comment]
Setup done
![image](https://user-images.githubusercontent.com/80741276/148509306-cb6ddcdf-27b1-4dbf-9e4e-19ce8068eedd.png)

**Comment by Joyan11** [comment]
<img width="436" alt="Screenshot 2022-01-07 at 1 10 55 PM" src="https://user-images.githubusercontent.com/23113259/148509431-1795cc70-a18e-4af6-8108-7bdc57af4393.png">

**Comment by nschalla** [comment]
![ReactNtaivesnippet](https://user-images.githubusercontent.com/36071265/148520370-2d7de8a7-201d-48f4-bf66-4d093642aac3.png)

**Comment by mohankumar80** [comment]
![Screenshot (10)](https://user-images.githubusercontent.com/50794740/148523401-23ab2fa1-9855-4675-ace2-ea3ec027dda1.png)

**Comment by jitendraag** [comment]
<img width="440" alt="Screenshot 2022-01-04 at 10 00 44 PM" src="https://user-images.githubusercontent.com/3295926/148523776-5714c765-0b21-463a-bf23-690390fdc57d.png">

Yes. :)

**Comment by akslucky619** [comment]
Donzo 
![Screenshot from 2022-01-07 15-18-40](https://user-images.githubusercontent.com/48199192/148525741-0d76105c-e46e-45ee-a956-08d0eec4a06b.png)
!

**Comment by jatin33** [comment]
Done with simulator setup and counter app
<img width="1792" alt="Screenshot 2022-01-07 at 3 19 51 PM" src="https://user-images.githubusercontent.com/13780467/148525790-19d33eb1-0eca-4778-ba16-5913203224e9.png">

**Comment by SaiKarthikYamavaram** [comment]
![RN-Initial-Setup](https://user-images.githubusercontent.com/37267369/148535066-655d409d-6f6f-4ef0-a336-fa545d341df3.png)

**Comment by palakgupta2712** [comment]
# Done with the setup!

<img width="263" alt="image" src="https://user-images.githubusercontent.com/61227144/148540218-83d1fb03-f587-4e85-9b63-4c13c9b84e08.png">

**Comment by vp5h** [comment]
- Done with Setup
- Tweet Ex
![image](https://user-images.githubusercontent.com/54497006/148543615-910e0cd4-077b-485e-a45a-feca1dd78c2f.png)

**Comment by dhanushkumar-dev** [comment]
![setupPostGithub](https://user-images.githubusercontent.com/79807326/148545705-95cdcd24-3933-4963-94e9-77df28a3d20e.PNG)
Done with setup
I'm using physical device and connecting via Vysor to view.
Couldn't run emulator due to minimum specs.

**Comment by shobit1337** [comment]
Done with assignment: 
![image](https://user-images.githubusercontent.com/37979367/148546905-7e4d27cb-9c5e-4bc8-9293-eaa50149f10c.png)

**Comment by avnkailash** [comment]
Done with the RN setup task

<img width="957" alt="rn_setup" src="https://user-images.githubusercontent.com/1806709/148548637-234e20f0-d6c6-42bb-af32-8575e2ff4578.png">

**Comment by rohan9896** [comment]
<img width="1280" alt="Screenshot 2022-01-07 at 9 32 57 PM" src="https://user-images.githubusercontent.com/56235204/148571214-27556c81-984c-4bff-89b6-0b836b5fbbe1.png">
Done with the setup with react native cli

**Comment by abhay05** [comment]
Done
![Screenshot_2022-01-04-23-35-17-944_com usergreeter](https://user-images.githubusercontent.com/23046900/148578141-ca4266a3-0b02-4fa9-98df-2d74e89f579b.jpg)

**Reply by sauravpathakofficial** [reply]
![image](https://user-images.githubusercontent.com/55847900/148580106-6a1eff2e-07e6-40e1-8a91-e64a8cbbab22.png)

**Comment by rayush158** [comment]
[
![Screenshot 2022-01-07 235507](https://user-images.githubusercontent.com/87794509/148589673-13124804-ef5e-4c77-a13d-a67bae7b586e.jpg)
](url)  done with setup and assignment also.

**Comment by emailvenky** [comment]
![RN_Setup](https://user-images.githubusercontent.com/1788643/148590109-5e444bd7-53b7-4167-a2b5-761c37eadbf9.png)

**Comment by rayush158** [comment]
Yes I completed that .

**Comment by Sahil-Bhatia** [comment]
Setup completed
![counterapp](https://user-images.githubusercontent.com/31650286/148593901-dc181aa2-1bbe-4461-b5d1-58223156dba2.PNG)

**Comment by vaishnavme** [comment]
<img width="1440" alt="react-native-desktop-setup" src="https://user-images.githubusercontent.com/42497931/148630887-5da23e46-86de-4bbb-b753-a999e8dd2b54.png">

**Comment by aakankshadhurandhar** [comment]
<img width="1440" alt="Screenshot 2022-01-04 at 9 30 26 PM" src="https://user-images.githubusercontent.com/34390765/148631536-eb77c2f1-61ee-489c-b1b3-8be070a61d65.png">

**Comment by asyriac** [comment]
Done with the setup.
![Capture](https://user-images.githubusercontent.com/43615822/148638948-7379fd91-55b4-433f-a234-dbab9c4eb785.PNG)

**Comment by Priyanka488** [comment]
got the setup done.

![image](https://user-images.githubusercontent.com/33698440/148650871-7e21559a-cad8-455a-ad7b-d681e92ea7c1.png)

**Comment by deepxcode** [comment]
Completed with the react-native setup and got the simulator working
![image](https://user-images.githubusercontent.com/31823246/148651129-42e1e715-2a8d-4d9a-a3dd-495d6642e8fc.png)

**Comment by Great-White-Shark-03** [comment]
Setup Complete!!
<img width="1440" alt="Screenshot 2022-01-08 at 10 59 47 PM" src="https://user-images.githubusercontent.com/68562518/148653796-19a332a0-9037-4da1-98f1-a74a47c28ab2.png">

**Comment by mohitshende** [comment]
Completed the setup and counter app.
![Screenshot (129)](https://user-images.githubusercontent.com/54403334/148670024-d0df3074-d75b-4b44-9100-9185de0d1aef.png)

**Comment by prachic17** [comment]
Done with the setup!! :)
![Screenshot from 2022-01-09 21-48-41](https://user-images.githubusercontent.com/61531836/148691040-71b2f9e0-45d1-47b1-95af-3171d688a6fe.png)

**Comment by chaitanyaj2222** [comment]
Here we go!
![output_320](https://user-images.githubusercontent.com/56331036/148725704-4bbb0d0e-e71b-47e0-ba0b-c5577c9288e5.png)

**Comment by Sumanta1998** [comment]
Done with the Setup
![image](https://user-images.githubusercontent.com/44142633/148732190-88de987f-5f1f-446e-9dc3-73473cad8df6.png)

**Comment by shaksham08** [comment]
Done with the setup
![image](https://user-images.githubusercontent.com/33754070/148785979-657fca4a-3b21-4117-9986-3d4857971f4c.png)

**Comment by Ramanpreet6262** [comment]
Done with setup and counter app.
https://github.com/Ramanpreet6262/Centauri-Counter-App

![image](https://user-images.githubusercontent.com/28749866/148823521-f8891864-ad46-47ea-b0b0-8f6ffa80b565.png)
